src/components/CameraPermissions/Primer/style.css

```diff
@@ -1,47 +1,52 @@
 @import (less) "../../Theme/constants.css";
 
-.text {
-  margin-top: 64px;
-  margin-bottom: 80px;
-}
+@font-size: 14;
+@font-size-units: @font-size*@unit;
+@sub-unit: 1em/@font-size;
```

Comment:
This is the first "hack" that's a bit weird. Just creating some `sub-unit`s on the fly, since we change the font-size and it confuses things

Reply:
Could you use a less mixin here to always make sure that unit returns the right value based on the actual font-size. And use the default value to calculate @font-size-units?

Reply:
I don't we can in LESS 😢